feat(content) Robotic Musical - New Culture Conversation (#6374)
endless-sky/endless-sky: data/hai/hai culture conversations.txt

			label end
			`	He takes your order diligently and with a smile.`
				decline

mission "Robotic Musical 1"
	minor
	source
		government "Hai"
		attributes "tourism"
		not attributes "station"
	to offer
		random < 1
	on offer
		conversation

> 			`Passing down the main thoroughfare of this spaceport you almost ignore a stack of haphazardly piled crates in the median strip. It's not uncommon for recently unloaded crates to temporarily sit in front of different businesses in a port, place conspicuously out of the way of the flow of foot traffic, but it is the disordered piling that seems out of place.`

Reviewer: ```suggestion
			`Passing down the main thoroughfare of this spaceport you almost ignore a stack of haphazardly piled crates in the median strip. It's not uncommon for recently unloaded crates to temporarily sit in front of different businesses in a port, placed conspicuously out of the way of the flow of foot traffic, but it is the disordered piling that seems out of place.`
```

> 			`	No sooner do you slow your step to give it your attention though, than the pile of crates immediately spring to life.`

Reviewer: ```suggestion
			`	No sooner do you slow your step to give it your attention though that the pile of crates immediately springs to life.`
```

Author: It would definitely be inappropriate to use "that" in this structure in Australian English. Is this some Americanism?

Reviewer 2: Original seems correct to me as well.

Reviewer: Huh. Guess I just haven't seen enough of that `No sooner` phasing, but Google agrees with you on that. I will stand by the `pile of crates` being a collective noun in this instance though (see convo on line 226).
```suggestion
			`	No sooner do you slow your step to give it your attention though, than the pile of crates immediately springs to life.`
```

			branch translator
				has "language: Wanderer"

> 			`	Music plays and some half dozen robotic caricatures of Hai in exaggerated dress spring forth from various crates and begin to caper about in a series of interactions presumably intended to be comedic, at least judging by how they keep turning away from each other to 'look' at the audience and wink. Holoprojected banners and glyphs, presumably advertisement information, flicker through a series of displays as it goes through a cycle before the entire cohort appear to finish up some kind of entreaty and immediately clamber back into the crates.`

Reviewer: ```suggestion
			`	Music plays and some half dozen robotic caricatures of Hai in exaggerated dress spring forth from various crates and begin to caper about in a series of interactions presumably intended to be comedic, at least judging by how they keep turning away from each other to "look" at the audience and wink. Holoprojected banners and glyphs, presumably advertisement information, flicker through a series of displays as it goes through a cycle before the entire cohort appears to finish up some kind of entreaty and immediately clamber back into the crates.`
```

Author: Pretty sure "appear" is correct as this is used in the context of a group noun.

https://dictionary.cambridge.org/grammar/british-grammar/appear

Reviewer: It's a [collective noun](https://www.touro.edu/departments/writing-center/tutorials/verb-agreement-with-collective-singular-nouns/). Basically, if the `cohort` acts as one collective group in "appearing to finish up", then you use singular verb (appears) instead of plural (appear). I suppose you could say that they're performing their action as individuals (in which case "appear" would be right), but it doesn't really feel like that to me, at least. If you'd prefer to keep it as-is though, I'm cool with letting it slide.
```suggestion
			`	Music plays and some half dozen robotic caricatures of Hai in exaggerated dress spring forth from various crates and begin to caper about in a series of interactions presumably intended to be comedic, at least judging by how they keep turning away from each other to "look" at the audience and wink. Holoprojected banners and glyphs, presumably advertisement information, flicker through a series of displays as it goes through a cycle before the entire cohort appear to finish up some kind of entreaty and immediately clamber back into the crates.`
```

Author: Well that was an interesting exercise in re-wiring my brain. I think the implication I was going for is that they'd divide their attention among onlookers, meaning they all either individually or in small groups are performing separate entreaties at various onlookers. Which would make it "appear"; however, I haven't carried across the particulars of that sufficiently well to make it clear that that is what is happening.

Reviewer: If that's the intent, then I'd agree that `appear` is right. I do think adding a sentence or phrase to clarify that this is performed as individuals rather than a group would be helpful, but I don't think it's absolutely necessary, especially for a culture mission. I'll leave it up to you to decide!

> 			`	Throughout this a handful of Hai have stopped to watch, and a couple of them let out small Hai chuckles at the conclusion before continuing on their way. Most of them simply roll their eyes and move on though. One child wanders too close as their parents steer around it and a little red holographic warning sign briefly flashes at them. No one seems keen to explain what you just saw and for the moment the crates remain resolutely closed.`

Reviewer 3: The phrase 'Hai chuckles' sounds weird to me, I think 'chuckles' is perfectly fine. 
```suggestion
			`	Throughout this a handful of Hai have stopped to watch, and a couple of them let out small chuckles at the conclusion before continuing on their way. Most of them simply roll their eyes and move on though. One child wanders too close as their parents steer around it and a little red holographic warning sign briefly flashes at them. No one seems keen to explain what you just saw and for the moment the crates remain resolutely closed.`
```

Reviewer: ```suggestion
			`	Throughout this a handful of Hai have stopped to watch, and a couple of them let out small chuckles at the conclusion before continuing on their way. Most of them simply roll their eyes and move on though. One child wanders too close as their parents steer around it and a little red holographic warning sign briefly flashes at them. No one seems keen to explain what you just saw, and for the moment the crates remain resolutely closed.`
```
Just needed to add a comma in.

			apply
				set roboticconfusion
			#This controls branching if there's a future follow-up where you get to see the performance.
			`	Now that you think about it, you're not sure they all climbed back into the same boxes they came out of.`
				decline
			label translator

> 			`	Music plays and some half dozen robotic caricatures of Hai in exaggerated dress spring forth from various crates and begin to caper about in a series of interactions. Your translator struggles to keep up as they run through a rapid-fire string of presumably witty wordplay jokes that don't translate well. The humor appears to be improvisational at first, but the illusion is so good that the realization of the impossibility of robots improvising only catches up with you after the fact. The entire thing is set to a musical score, with beat rests at various points where the animatronic Hai keep turning away from each other to 'look' at the audience and wink. Holoprojected banners and glyphs flicker through a series of displays informing you that this is an advertisement for a fully robotic comedy musical called "The Grandmother's Rules". It claims to be complete with interactive heckling responses and provides a list of dates and venues, all in the future, where it will be showing.`

Reviewer 3: Don't really think this phrase is necessary.
```suggestion
			`	Music plays and some half dozen robotic caricatures of Hai in exaggerated dress spring forth from various crates and begin to caper about in a series of interactions. Your translator struggles to keep up as they run through a rapid-fire string of presumably witty wordplay jokes that don't translate well. The humor appears to be improvisational at first, but the illusion is so good that the realization of the impossibility of robots improvising only catches up with you after the fact. The entire thing is set to a musical score, with beat rests at various points where the animatronic Hai keep turning away from each other to 'look' at the audience and wink. Holoprojected banners and glyphs flicker through a series of displays informing you that this is an advertisement for a fully robotic comedy musical called "The Grandmother's Rules". It claims to be complete with interactive heckling responses and provides a list of dates and venues where it will be showing.`
```

> 			`	Music plays and some half dozen robotic caricatures of Hai in exaggerated dress spring forth from various crates and begin to caper about in a series of interactions. Your translator struggles to keep up  as they run through a rapid-fire string of presumably witty wordplay jokes that don't translate well and apparently improvisational humor. You realize that of course it can't actually be improvisational, but the illusion is good enough that the realization only catches up with you after the fact. The entire thing is set to a musical score, with beat rests at various points where the animatronic Hai keep turning away from each other to 'look' at the audience and wink. Holoprojected banners and glyphs flicker through a series of displays informing you that this is an advertisement for a fully robotic comedy musical called "The Grandmother's Rules". It claims to be complete with interactive heckling responses and provides a list of dates and venues, all in the future, where it will be showing.`

Reviewer 3: ```suggestion
			`	Music plays and some half dozen robotic caricatures of Hai in exaggerated dress spring forth from various crates and begin to caper about in a series of interactions. Your translator struggles to keep up as they run through a rapid-fire string of presumably witty wordplay jokes that don't translate well. The humor appears to be improvisational at first, but the illusion is so good that the realization of the impossibility of robots improvising only catches up with you after the fact. The entire thing is set to a musical score, with beat rests at various points where the animatronic Hai keep turning away from each other to 'look' at the audience and wink. Holoprojected banners and glyphs flicker through a series of displays informing you that this is an advertisement for a fully robotic comedy musical called "The Grandmother's Rules". It claims to be complete with interactive heckling responses and provides a list of dates and venues, all in the future, where it will be showing.`
```

> 			`	At the end each one of the robots strikes a pose and invites a different demographic to come along and "have the time of a decade", with ticket prices starting from 73 credits, before the entire cohort finish up and immediately clamber back into the crates. You're not sure they all climbed back into the same boxes they came out of.`

Reviewer: ```suggestion
			`	At the end each one of the robots strikes a pose and invites a different demographic to come along and "have the time of a decade," with ticket prices starting from 73 credits, before the entire cohort finish up and immediately clamber back into the crates. You're not sure they all climbed back into the same boxes they came out of.`
```

> 			`	Throughout this a handful of Hai have stopped to watch, and a couple of them let out small Hai chuckles at the conclusion before continuing on their way. Most of them simply roll their eyes and move on though.`

Reviewer: ```suggestion
			`	Throughout this a handful of Hai have stopped to watch, and a couple of them let out small chuckles at the conclusion before continuing on their way. Most of them simply roll their eyes and move on though.`
```

> 			`	One middle aged Hai in the crowd with racoon-like grey around his eyes wears the type of attire typical of their merchant captains. He eyes you curiously and steps over to you. "Well well Captain, that's a remarkable box you have there. Didn't know they'd started making live translator devices! Did it keep up alright?"`

Reviewer 3: ```suggestion
			`	One middle aged Hai in the crowd with racoon-like grey around his eyes wears the type of attire typical of their merchant captains. He eyes you curiously and steps over to you. "Well, Captain, that's a remarkable box you have there. Didn't know they'd started making live translator devices! Did it keep up alright?"`
```

Reviewer 3: This last review was just some nitpicks, if you really feel strongly about it then you can ignore it.

> 			`	One middle aged Hai with racoon-like grey around his eyes and wearing the type of attire typical of their merchant captains eyes you curiously and steps over to you. "Well well Captain, that's a remarkable box you have there. Didn't know they'd started making live translator devices! Did it keep up alright?"`

Reviewer 3: ```suggestion
			`	One middle aged Hai in the crowd with racoon-like grey around his eyes wears the type of attire typical of their merchant captains. He eyes you curiously and steps over to you. "Well well Captain, that's a remarkable box you have there. Didn't know they'd started making live translator devices! Did it keep up alright?"`
```

			choice

> 				`	"Oh it's the first of its kind."`

Reviewer 3: ```suggestion
				`	"Oh, it's the first of its kind."`
```

					goto first
				`	"I think I got the gist of it, though it only reads things out if I look directly at them."`

> 			`	"Ah yes, I suppose that's a logical feature," he says. "I can only imagine it would be quite unwieldy if it tried to read out every piece of text in its vicinity.`

Reviewer: ```suggestion
			`	"Ah yes, I suppose that's a logical feature," he says. "I can only imagine it would be quite unwieldy if it tried to read out every conversation in its vicinity.`
```

Author: This is referring to when it translates text not spoken word using its cameras to identify context and content.

Reviewer: Gotcha. Feel free to resolve!

				goto resolve
			label first

> 			`	"The first?" He studies you for a second. "Oh you're Captain <last>! What a pleasure to meet you."`

Reviewer 3: ```suggestion
			`	"The first?" He studies you for a second. "Oh, you're Captain <last>! What a pleasure to meet you."`
```

> 			`	He looks you up and down with a wry grin, "Somehow you're shorter than I expected.`

Reviewer: ```suggestion
			`	He looks you up and down with a wry grin. "Somehow you're shorter than I expected.`
```

			label resolve
			`	"Well if you do get the chance, I do recommend the performance. I saw it a few months back and it really is quite good. Particularly popular on our worlds with many humans."`

> 			`	He flicks you a friendly wave, "I must be off, don't get too close to the displays, they don't like being touched!"`

Reviewer 3: ```suggestion
			`	He flicks you a friendly wave. "I must be off. Don't get too close to the displays, they don't like being touched!"`
```

> 			`	The Hai's sounds very pleased with himself as he ambles off, quietly chortling to himself as he departs.`

Reviewer 3: ```suggestion
			`	The Hai sounds very pleased with himself as he ambles off, quietly chortling to himself as he departs.`
```

Reviewer: ```suggestion
			`	The Hai's sounds very pleased with himself as he ambles off, quietly chortling as he departs.`
```
Just wanted to get rid of the double `to himself`.

Author: ```suggestion
			`	The Hai sounds very pleased as he ambles off, quietly chortling to himself as he departs.`
```
Removed the alternate reference to `himself` and incorporated roadrunner's change.
